.github/workflows/main.yml

```diff
@@ -2,6 +2,11 @@ name: CI
 
 on: [push, pull_request]
 
+env:
+  GITHUB_BASE_SHA: ${{ github.event.pull_request.base.sha }}
+  GITHUB_HEAD_SHA: ${{ github.event.pull_request.head.sha }}
+  GITHUB_PULL_REQUEST: ${{ github.event.number }}
+
```

Comment:
Can you add a comment saying where these are used?

Reply:
Yup, they're used for translating Travis environment variables, I'll update them.